src/modules/scene/add_question_scene.py

```diff
@@ -1,10 +1,12 @@
+import sys
```

Comment:
Assuming this file is not completed at the time of PR, so I will skip review for now